src/shared/components/palette/multi-select-list/write-multi-select-list-field.component.ts

```diff
@@ -41,7 +40,8 @@ export class WriteMultiSelectListFieldComponent extends AbstractFieldWriteCompon
   }
 
   isSelected(code): AbstractControl {
-    return this.checkboxes.controls.find(control => control.value === code);
+    if (this.checkboxes && this.checkboxes.controls) {
+      return this.checkboxes.controls.find(control => control.value === code);
```

Comment:
Just wanted to check that `find()` will be transpiled correctly for IE11, since `find()` isn't supported for that browser

Reply:
This was already in place. We simply made sure that `this.checkboxes.controls` wasn't null.